### PR TITLE
task: adding-kit-db-reset

### PR DESCRIPTION
**Pull request by MateoCaicedoW**
## Description ##
This PR adds the ability to reset the db using the `kit db reset` command.

### CODE REVIEW

File: kit/database.go [hunk 1]
```diff
@@ -78,6 +78,34 @@ func database(args []string) error {
 		}
 
 		fmt.Println("✅ Database dropped successfully")
+
+	case "reset":
+		err := db.Drop(url)
+		if err != nil {
```
Comment (paganotoni) [inline kit/database.go:84]: I wonder if we should ignore the error on Drop. Would it err if the DB is not there?

File: kit/database.go [hunk 2]
```diff
@@ -78,6 +78,34 @@ func database(args []string) error {
...
+			return err
+		}
+
+		err = db.Create(url)
+		if err != nil {
+			return err
```
Comment (paganotoni) [inline kit/database.go:90]: @MateoCaicedoW can we wrap these errors so the message explains better what happened?

Comment (MateoCaicedoW) [reply kit/database.go:90]: sure!

File: kit/database.go (hunk 1 above)
Comment (MateoCaicedoW) [reply kit/database.go:84]: Oh! I just noticed the sqlite and postgres implementations don't return an error if there's no database :D

**Comment by paganotoni** [comment]
@MateoCaicedoW we still need to add some tests :)

File: kit/database_test.go [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+		// Set the DATABASE_URL to the temporary file
+		os.Setenv("DATABASE_URL", "test.db")
+		defer os.Remove("test.db")
+		cmd := exec.Command("go", "run", ".", "db", "create")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			t.Fatalf("error running command: %v", err)
+		}
+
+		if string(out) != "✅ Database created successfully\n" {
```
Comment (paganotoni) [inline kit/database_test.go:78]: @MateoCaicedoW check the file is created for SQLite.

File: kit/database_test.go [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+		// Set the DATABASE_URL to the temporary file
+		os.Setenv("DATABASE_URL", "test.db")
+		defer os.Remove("test.db")
+		cmd := exec.Command("go", "run", ".", "db", "drop")
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			t.Fatalf("error running command: %v", err)
+		}
+
+		if string(out) != "✅ Database dropped successfully\n" {
```
Comment (paganotoni) [inline kit/database_test.go:96]: @MateoCaicedoW check the file is not there.

**Review by paganotoni — CHANGES_REQUESTED** [review]
@MateoCaicedoW need to check the files.

File: kit/database_test.go (hunk 3 above)
Comment (MateoCaicedoW) [reply kit/database_test.go:78]: what do you mean?

File: kit/database_test.go (hunk 4 above)
Comment (MateoCaicedoW) [reply kit/database_test.go:96]: When the file does not exist, it returns nil

**Pull request merged**